mssqlcli/mssql_cli.py

```diff
@@ -366,6 +368,20 @@ def execute_query(self, text):
             output = self._execute_interactive_command(text)
         else:
             # non-interactive mode
+            if self.input_file:
+                if text:
+                    # return error if both query text and an input file are specified
+                    click.secho("Invalid arguments: either -Q or -i may be specified.",
```

Comment:
Should you use the full names of these arguments in the error message? What if someone doesn't know what -i or -Q means?